### DISCUSSION TITLE
Add constant framerate option for more smooth bitrate and pacing, solving issue 1877(which at least is problematic for NVENC encoder)

### DISCUSSION THREAD

**Discussion by Originalimoc** [LizardByte/community#401] [Sunshine] [answered]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your issue described in the documentation?

- [X] I have read the documentation

### Is your issue present in the nightly release?

- [X] This issue is present in the nightly release

### Describe the Bug

If window duplicate producing less frame than asked framerate, send last available frame to the encoder, capped by moonlight request FPS or host current monitor Hz.

### Host Operating System

Windows

### Operating System Version

11

### Architecture

64 bit

### Sunshine commit or version

529f1b8

### Package

Windows - installer

### GPU Type

Nvidia

### GPU Model

3xxx Desktop + AMD CPU

### GPU Driver/Mesa Version

551.61 Studio

### Config

```Shell
Default install
```

### Relevant log output

```shell
Not Relevant. No error log.
```

**Comment by cgutman** [comment, marked as answer]
This should be supported after #2149 is merged.

**Comment by Originalimoc** [comment]
Huh interesting coincidence. But wait why can I capture Game Bar and Lock screen even now?
